pkg/cloudhypervisor/client.go

```diff
@@ -0,0 +1,440 @@
+package cloudhypervisor
+
+import (
+	"context"
+	"net"
+	"net/http"
+
+	"github.com/carlmjohnson/requests"
+)
+
+const (
+	DefaultServerEndpoint = "http://localhost/api/v1/"
+)
+
+// Client represents a client for the Cloud Hypervisor API.
+type Client interface {
+	// VmmPing checks for API server availability.
+	VmmPing(ctx context.Context) (*VmmPingResponse, error)
+	// VmmShutdown shuts down the cloud-hypervisor vmm.
+	VmmShutdown(ctx context.Context) error
+	// Info returns general information about the cloud-hypervisor Virtual Machine (VM) instance.
+	Info(ctx context.Context) (*VmInfo, error)
+	//Counters(ctx context.Context) *VmmCo
```

Comment:
what dis?

Reply:
Ah, thats a future todo that i forgot to remove. I have removed it now :)